client/components/add-new-site/modals/index.tsx

```diff
@@ -0,0 +1,19 @@
+import { useMemo } from 'react';
+import AsyncLoad from 'calypso/components/async-load';
+import isA8CForAgencies from 'calypso/lib/a8c-for-agencies/is-a8c-for-agencies';
+
+const AddNewSiteModals = () => {
+	const renderContent = useMemo( () => {
+		switch ( true ) {
+			case isA8CForAgencies():
+				return (
+					<AsyncLoad require="calypso/components/add-new-site/modals/a4a" placeholder={ null } />
```

Comment:
Optional: feel free to leave async loading for later, unless you're certain it will have a big impact.

Reply:
Yeah. I realized that this would require some thinking. So, I removed it [here](https://github.com/Automattic/wp-calypso/pull/97499/commits/8365a989faeeda3c1d128f37813e2595c66f08de).

Reply:
Yep, I agree - introducing async loading early can help address some of @jkguidaven's concerns.